### PR TITLE
docs: update usage restriction for podchaos

### PR DESCRIPTION
**Pull request by STRRL**
Signed-off-by: STRRL <im@strrl.dev>

close https://github.com/chaos-mesh/website/issues/231

### CODE REVIEW

**Comment by STRRL** [comment]
PTAL

/cc @YangKeao @cwen0

**Review by cwen0 — APPROVED** [review]
LGTM

File: i18n/zh/docusaurus-plugin-content-docs/current/simulate-pod-chaos-on-kubernetes.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ PodChaos 是 Chaos Mesh 中的一种故障类型，通过创建 PodChaos 类型
 
 ## 使用限制
 
-目前 Chaos Mesh 仅支持向特定类型的 Pod 中注入故障，例如 Deployment、Statefulset、Daemonset。Chaos Mesh 不支持向独立的 Pod 中注入故障，独立的 Pod 指未绑定到 ReplicaSet 或 Deployment 的 Pod。
+Chaos Mesh 可以向任意 Pod 注入 PodChaos，无论是否绑定到 Deployment、StatefulSet、DaemonSet 或其他控制器。但是在向独立的 Pod 注入 PodChaos 时，可能会有不同的预期。例如，当我们向独立的 Pod 注入 "pod-kill" 混沌时，无法保证应用程序能够在故障发生后恢复正常。
```
Comment (en-jin19) [inline i18n/zh/docusaurus-plugin-content-docs/current/simulate-pod-chaos-on-kubernetes.md:17]:
```suggestion
Chaos Mesh 可以向任一 Pod 注入 PodChaos，无论其 Pod 是否绑定至 Deployment，StatefulSet，DaemonSet 或其他控制器。然而，当向独立的 Pod 注入 PodChaos 时，可能会引起不同的情况。比如，向独立的 Pod 注入 "pod-kill" 混沌时，无法保证应用程序在故障发生后能够恢复正常。
```

File: i18n/zh/docusaurus-plugin-content-docs/version-2.0.7/simulate-pod-chaos-on-kubernetes.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ PodChaos 是 Chaos Mesh 中的一种故障类型，通过创建 PodChaos 类型
 
 ## 使用限制
 
-目前 Chaos Mesh 仅支持向特定类型的 Pod 中注入故障，例如 Deployment、Statefulset、Daemonset。Chaos Mesh 不支持向独立的 Pod 中注入故障，独立的 Pod 指未绑定到 ReplicaSet 或 Deployment 的 Pod。
+Chaos Mesh 可以向任意 Pod 注入 PodChaos，无论是否绑定到 Deployment、StatefulSet、DaemonSet 或其他控制器。但是在向独立的 Pod 注入 PodChaos 时，可能会有不同的预期。例如，当我们向独立的 Pod 注入 "pod-kill" 混沌时，无法保证应用程序能够在故障发生后恢复正常。
```
Comment (en-jin19) [inline i18n/zh/docusaurus-plugin-content-docs/version-2.0.7/simulate-pod-chaos-on-kubernetes.md:17]:
```suggestion
Chaos Mesh 可以向任一 Pod 注入 PodChaos，无论其 Pod 是否绑定至 Deployment，StatefulSet，DaemonSet 或其他控制器。然而，当向独立的 Pod 注入 PodChaos 时，可能会引起不同的情况。比如，向独立的 Pod 注入 "pod-kill" 混沌时，无法保证应用程序在故障发生后能够恢复正常。
```

File: i18n/zh/docusaurus-plugin-content-docs/version-2.1.5/simulate-pod-chaos-on-kubernetes.md [hunk 3]
```diff
@@ -14,7 +14,7 @@ PodChaos 是 Chaos Mesh 中的一种故障类型，通过创建 PodChaos 类型
 
 ## 使用限制
 
-目前 Chaos Mesh 仅支持向特定类型的 Pod 中注入故障，例如 Deployment、Statefulset、Daemonset。Chaos Mesh 不支持向独立的 Pod 中注入故障，独立的 Pod 指未绑定到 ReplicaSet 或 Deployment 的 Pod。
+Chaos Mesh 可以向任意 Pod 注入 PodChaos，无论是否绑定到 Deployment、StatefulSet、DaemonSet 或其他控制器。但是在向独立的 Pod 注入 PodChaos 时，可能会有不同的预期。例如，当我们向独立的 Pod 注入 "pod-kill" 混沌时，无法保证应用程序能够在故障发生后恢复正常。
```
Comment (en-jin19) [inline i18n/zh/docusaurus-plugin-content-docs/version-2.1.5/simulate-pod-chaos-on-kubernetes.md:17]:
```suggestion
Chaos Mesh 可以向任一 Pod 注入 PodChaos，无论其 Pod 是否绑定至 Deployment，StatefulSet，DaemonSet 或其他控制器。然而，当向独立的 Pod 注入 PodChaos 时，可能会引起不同的情况。比如，向独立的 Pod 注入 "pod-kill" 混沌时，无法保证应用程序在故障发生后能够恢复正常。
```

File: versioned_docs/version-2.0.7/simulate-pod-chaos-on-kubernetes.md [hunk 4]
```diff
@@ -14,7 +14,7 @@ PodChaos is a fault type in Chaos Mesh. By creating a PodChaos experiment, you c
 
 ## Usage restrictions
 
-Currently, Chaos Mesh only supports fault injection to certain types of Pod, such as Deployment, StatefulSet, and DaemonSet. Chaos Mesh does not support injecting faults into an independent Pod. An independent Pod means a Pod that is not bound to ReplicaSet or Deployment Pod.
+Chaos Mesh could inject PodChaos into any Pod, no matter whether it is bound to Deployment, StatefulSet, DaemonSet, or other controllers. But there might be different expected situations when you inject PodChaos into an independent Pod. For example, when we inject "pod-kill" chaos into an independent Pod, there is no way to guarantee the application would recover from the failure.
```
Comment (en-jin19) [inline versioned_docs/version-2.0.7/simulate-pod-chaos-on-kubernetes.md:17]:
```suggestion
Chaos Mesh can inject PodChaos into any Pod, no matter whether the Pod is bound to Deployment, StatefulSet, DaemonSet, or other controllers. However, when you inject PodChaos into an independent Pod, some different situations might occur. For example, when you inject "pod-kill" chaos into an independent Pod, Chaos Mesh cannot guarantee that the application recovers from its failure.
```

File: versioned_docs/version-2.1.5/simulate-pod-chaos-on-kubernetes.md [hunk 5]
```diff
@@ -14,7 +14,7 @@ PodChaos is a fault type in Chaos Mesh. By creating a PodChaos experiment, you c
 
 ## Usage restrictions
 
-Currently, Chaos Mesh only supports fault injection to certain types of Pod, such as Deployment, StatefulSet, and DaemonSet. Chaos Mesh does not support injecting faults into an independent Pod. An independent Pod means a Pod that is not bound to ReplicaSet or Deployment Pod.
+Chaos Mesh could inject PodChaos into any Pod, no matter whether it is bound to Deployment, StatefulSet, DaemonSet, or other controllers. But there might be different expected situations when you inject PodChaos into an independent Pod. For example, when we inject "pod-kill" chaos into an independent Pod, there is no way to guarantee the application would recover from the failure.
```
Comment (en-jin19) [inline versioned_docs/version-2.1.5/simulate-pod-chaos-on-kubernetes.md:17]:
```suggestion
Chaos Mesh can inject PodChaos into any Pod, no matter whether the Pod is bound to Deployment, StatefulSet, DaemonSet, or other controllers. However, when you inject PodChaos into an independent Pod, some different situations might occur. For example, when you inject "pod-kill" chaos into an independent Pod, Chaos Mesh cannot guarantee that the application recovers from its failure.
```

File: docs/simulate-pod-chaos-on-kubernetes.md [hunk 6]
```diff
@@ -14,7 +14,7 @@ PodChaos is a fault type in Chaos Mesh. By creating a PodChaos experiment, you c
 
 ## Usage restrictions
 
-Currently, Chaos Mesh only supports fault injection to certain types of Pod, such as Deployment, StatefulSet, and DaemonSet. Chaos Mesh does not support injecting faults into an independent Pod. An independent Pod means a Pod that is not bound to ReplicaSet or Deployment Pod.
+Chaos Mesh could inject PodChaos into any Pod, no matter whether it is bound to Deployment, StatefulSet, DaemonSet, or other controllers. But there might be different expected situations when you inject PodChaos into an independent Pod. For example, when we inject "pod-kill" chaos into an independent Pod, there is no way to guarantee the application would recover from the failure.
```
Comment (en-jin19) [inline docs/simulate-pod-chaos-on-kubernetes.md:17]:
```suggestion
Chaos Mesh can inject PodChaos into any Pod, no matter whether the Pod is bound to Deployment, StatefulSet, DaemonSet, or other controllers. However, when you inject PodChaos into an independent Pod, some different situations might occur. For example, when you inject "pod-kill" chaos into an independent Pod, Chaos Mesh cannot guarantee that the application recovers from its failure.
```

**Comment by STRRL** [comment]
Updated! PTAL @en-jin19

**Review by Oreoxmt — APPROVED** [review]

**Review by YangKeao — APPROVED** [review]
LGTM

**Pull request merged**